data/0.95.0/schema/election_constants.schema.json

```diff
@@ -1,5 +1,4 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
We want to define the schema in ALL of these files. The schema shifts frequently and it can check the json format.